src/event_loop.rs

```diff
@@ -253,6 +253,20 @@ impl<T> EventLoopWindowTarget<T> {
   pub fn cursor_position(&self) -> Result<PhysicalPosition<f64>, ExternalError> {
     self.p.cursor_position()
   }
+
+  /// Sets the progress bar state
+  /// 
+  /// ## Platform-specific
+  /// 
+  /// - **Windows:** Use the Progress Bar Function Available in Window (Windows can have different progress bars for different window)
+  /// - **Linux:** Only supported desktop environments with `libunity` (e.g. GNOME).
+  /// - **macOS:** Unimplemented.
+  /// - **iOS / Android:** Unsupported.
+  #[inline]
+  pub fn set_progress_bar(&self, progress: ProgressBarState) {
+    #[cfg(target_os = "linux")]
+    self.p.set_progress_bar(progress)
```

Comment:
```suggestion
  pub fn set_progress_bar(&self, _progress: ProgressBarState) {
    #[cfg(target_os = "linux")]
    self.p.set_progress_bar(_progress)
```

Oops, this should have a `_` prefix, my fault 😿